django/applications/catmaid/static/js/widgets/3dviewer.js

```diff
@@ -23,6 +23,12 @@
    * compute with web workers (betweenness centrality shading). */
   var WebGLApplication = function() {
     this.widgetID = this.registerInstance();
+
+    if (CATMAID.MAX_WEBGL_CONTEXTS - CATMAID.countWebGlContexts() < 1) {
+      CATMAID.TooManyWebGlContextsDialog().show();
```

Comment:
That sounds likely. Can you chain the `.show()` straight off the new instance or do you need to assign it to a variable?